Regions/Ausbaxter_Load region render matrix slot 1.lua

```diff
@@ -0,0 +1,128 @@
+--[[
+ * ReaScript Name: Load region render matrix slot n
+ * Description: Allows quick storage and recall of reaper render matrix.
+ * Instructions: Make track selections in region matrix, use the "complementary Save region render matrix slot n" to save the setup. 
+      Use the 'load' scripts to recall.
+ * Author: Ausbaxter
+ * Author URI: https://forum.cockos.com/member.php?u=107072
+ * Repository: GitHub > Ausbaxter
+ * Repository URL: https://github.com/ausbaxter/Reascripts
+ * File URI: https://github.com/ausbaxter/Reascripts/Region Render Matrix/Load region render matrix slot 1
+ * Licence: GPL v3
+ * REAPER: 5.xx
+ * Extensions: SWS Extension
+ * Version: 1.0
+--]]
+ 
+--[[
+ * Changelog:
+ * v1.0 (2018-01-09)
+  + Initial Release
+--]]
+--------------------User Area---------------------------------------------------------------------------
+--To create more save slots duplicate this script and change the slot number variable
+
+slot_number = 1 --only use integers
```

Comment:
It would probably be better to bundle all these slots and actions and into a single installable package. I think there is no point in allowing users to install only "save slot 4" without the others actions at the same time.

By making a single package you will be able to reuse the same code for each slot without copy-pasting on your end. It also guarantees that all slots are installed and updated together.

See [multiple slots](https://github.com/cfillion/reapack-index/wiki/Examples#multiple-slots) in the examples page.

Using this technique you can have down to just one single file to upload here.